src/mappings/mappingHandlers.ts

```diff
@@ -138,32 +199,28 @@ export async function handleEvent(event: SubstrateEvent): Promise<void> {
     })
   );
 
-  await Event.create({
-    id: `${blockId}/${eventIdx}`,
+  await Promise.all(handlerPromises);
+}
+
+// handles calls to populate native GraphQL tables as well as what is needed for tooling
```

Comment:
When I get the asset handler migrated, this can be reverted to just one handler, so only events is weird